events/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.gis.db import models as model
+
+
+class Category(models.Model):
+    name = models.CharField(max_length=255)
+    
+    parent = models.ForeignKey(
+        "events.Category", 
+        related_name="categories", 
+        null=True, 
+        blank=True, 
+        on_delete=models.CASCADE
+    )
+
+
+class Event(models.Model):
+    class EventType(models.TextChoices):
```

Comment:
we usually move these to separate choices.py file otherwise it's very difficult to access these choices